packages/core/src/login.ts

```diff
@@ -110,8 +112,8 @@ export async function loginSilently(
     if (tokens !== null) {
       return tokens;
     }
-  } catch (_) {
-    // don't do anything.
+  } catch (error) {
+    logger.log(error);
```

Comment:
maybe we should make logger as a singleton? so we don't need to pass it here at all?
same in constructor.
(it's a suggestion, I don't have strong preferences here, can be this way as well I guess)

Reply:
yep, that's a good point, redone as a singleton